lib/ansible/modules/monitoring/zabbix_group.py

```diff
@@ -195,7 +201,8 @@ def main():
 
     # login to zabbix
     try:
-        zbx = ZabbixAPI(server_url, timeout=timeout, user=http_login_user, passwd=http_login_password)
+        zbx = ZabbixAPI(server_url, timeout=timeout, user=http_login_user, passwd=http_login_password,
+                        validate_certs=validate_certs)
```

Comment:
I suspect there is a problem with underlaying zabbix-api when pulled from pypi.python.org. Package (created Feb 13 2015) seems to be not updated with the current state of repository. Skipping cert validation was introduced later in this commit https://github.com/gescheit/scripts/commit/4456158e6a56dd52f23a803c08f280ffa6791477 from Nov 19 2015